wasm/main.go

```diff
@@ -104,12 +98,17 @@ func render() {
 		return
 	}
 
-	allRendered := ""
-	for _, v := range rendered {
-		allRendered += v
+	manifests, notes := "", ""
+	for k, v := range rendered {
+		if strings.HasSuffix(k, "NOTES.txt") {
```

Comment:
Just out of curiosity, what was `k` prefixed with ?
Why can’t we just do `if k == "NOTES.txt" {` ?

Reply:
`k` is the whole file path